api/v1/verticadb_webhook.go

```diff
@@ -1153,6 +1169,57 @@ func (v *VerticaDB) hasValidTLSModes(allErrs field.ErrorList) field.ErrorList {
 	return allErrs
 }
 
+// hasValidTLSModes checks whether the TLS version and cipher suites are valid
+func (v *VerticaDB) hasValidDBTLSConfig(allErrs field.ErrorList) field.ErrorList {
+	if !v.UseVClusterOpsDeployment() && v.Spec.DBTLSConfig != nil {
```

Comment:
I think you also want to check if TLS is enabled:

```suggestion
	if !v.UseVClusterOpsDeployment() && v.Spec.DBTLSConfig != nil {
```
```suggestion
	if !v.UseVClusterOpsDeployment() && !vmeta.UseTLSAuth(v.Annotations) && v.Spec.DBTLSConfig != nil {
```

Reply:
The reconciler will start to reconcile tls version and cipher suites when tls is enabled and configured. This rule controls when users can add dbTlsConfig block to their yaml file. It is reasonable to assume they will add it after tls is enabled.